core/templates/pages/exploration-editor-page/modal-templates/exploration-metadata-modal.template.html

```diff
@@ -15,34 +15,39 @@ <h3>Add Some Details</h3>
                                 form-style="{'font-weight': 'normal'}">
   </exploration-objective-editor>
   <p ng-if="requireCategoryToBeSpecified">
-    Add a category: how would you categorize your exploration?
-    <select2-dropdown item="explorationCategoryService.displayed"
-                      choices="CATEGORY_LIST_FOR_SELECT2"
-                      placeholder="Choose or type new"
-                      new-choice-regex="^[A-Z a-z]+$"
-                      width="100%"
-                      invalid-search-term-message="Invalid category name">
-    </select2-dropdown>
+    <label ng-style="{'font-weight':'normal'}"> Add a category: How would you categorize your exploration?
```

Comment:
Instead of using inline style, can we use the bootstrap class `font-weight-normal` here and below (see [example](https://getbootstrap.com/docs/4.0/utilities/text/#font-weight-and-italics))?